web/themes/joinup/joinup_theme.theme

```diff
@@ -1599,3 +1606,41 @@ function joinup_theme_form_views_exposed_form_alter(&$form, FormStateInterface $
     $form['keys']['#attributes']['placeholder'] = t('Start typing to search...');
   }
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ *
+ * Provide a separate suggestion for menus depending on their region.
+ */
+function joinup_theme_theme_suggestions_menu_alter(array &$suggestions, array $variables) {
+  if (isset($variables['attributes']['block'])) {
+    $hook = str_replace(['block-', '-'], ['', '_'], $variables['attributes']['block']);
+    $block = Block::load($hook);
+    $region = $block->getRegion();
+    $suggestions[] = $variables['theme_hook_original'] . '__' . $region;
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Remove the default links from the table of contents.
+ */
+function joinup_theme_preprocess_menu__og__navigation__content_bottom(array &$variables) {
+  foreach ($variables['items'] as $menu_link_id => $menu_item) {
+    /** @var \Drupal\Core\StringTranslation\TranslatableMarkup $title */
+    $title = $menu_item['title'];
+    /** @var \Drupal\Core\Url $url */
+    $url = $menu_item['url'];
+
+    if ($title == 'Overview' && $url->getRouteName() === 'entity.rdf_entity.canonical') {
```

Comment:
The strict comparison is not used as some link titles are strings but some are of class `TranslatableMarkup`. While all of the default links have titles of class `TranslatableMarkup`, we would need to check the instance so that we don't call a method on a string in case it is not an object and so on and so on and I don't think that this would require less processing power than simply checking with a double quote.